add support for css `@property` (#1092)
vanilla-extract-css/vanilla-extract: .changeset/angry-rules-tease.md

---
"@vanilla-extract/babel-plugin-debug-ids": minor
---

> Add support for `createVar` calls with property definitions

Reviewer: ```suggestion
Add support for `createVar` calls that declare `@property` rules
```
